libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/components/media/FakeWaveformFactory.kt

```diff
@@ -20,17 +20,15 @@ import kotlinx.collections.immutable.ImmutableList
 import kotlinx.collections.immutable.toPersistentList
 import kotlin.random.Random
 
-object FakeWaveformFactory {
```

Comment:
Unrelated to the PR but addresses https://github.com/vector-im/element-x-android/pull/1661#discussion_r1374419884